src/components/visualizations/utils/drilldownEventing.ts

```diff
@@ -186,6 +188,19 @@ const tickLabelClickDebounce = debounce(
     },
 );
 
+function sanitizeContextPoints(
+    chartType: ChartType,
+    points: IHighchartsPointObject[],
+): IHighchartsPointObject[] {
+    if (chartType !== VisualizationTypes.HEATMAP) {
```

Comment:
should reuse [isHeatmap](https://github.com/gooddata/gooddata-react-components/blob/next-7.1.0/src/components/visualizations/utils/common.ts#L73) function

Reply:
updated